exporter/datadogexporter/internal/clientutil/http.go

```diff
@@ -29,8 +30,14 @@ var (
 
 // NewHTTPClient returns a http.Client configured with a subset of the confighttp.ClientConfig options.
 func NewHTTPClient(hcs confighttp.ClientConfig) *http.Client {
+	// If the ProxyURL field in the configuration is set, the HTTP client will use the proxy.
+	// Otherwise, the HTTP client will use the system's proxy settings.
+	httpProxy := http.ProxyFromEnvironment
+	if parsedProxyURL, err := url.Parse(hcs.ProxyURL); err == nil && parsedProxyURL.Scheme != "" {
```

Comment:
Can you add a test to https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/internal/clientutil/http_test.go#L28?